src/main/groovy/org/transmartproject/batch/highdim/mrna/platform/MrnaAnnotationRowValidator.groovy

```diff
@@ -32,19 +24,7 @@ class MrnaAnnotationRowValidator implements Validator {
     void validate(Object target, Errors errors) {
         assert target instanceof MrnaAnnotationRow
 
-        /* platformObject.id should be normalized to uppercase because it comes
-         * from the normalized parameter PLATFORM, but it may be that platform
-         * name in the data file is not */
-        if (target.gplId.toUpperCase(locale) != platformObject.id) {
-            errors.rejectValue 'gplId', 'expectedConstant',
-                    [platformObject.id, target.gplId] as Object[], null
-        }
-
-        if (!platformObject.organism.equalsIgnoreCase(target.organism)) {
-            errors.rejectValue 'organism', 'expectedConstant',
-                    [platformObject.organism, target.organism] as Object[],
-                    null
-        }
+        PlatformOrganismValidator.super.validate(target, errors)
```

Comment:
@cataphract How the Groovy way would look like? Could you maybe point me to a documentation?